src/fragments/lib/auth/native_common/signin_with_custom_flow/common.mdx

```diff
@@ -1,49 +1,71 @@
-The Auth category can be configured to perform a custom authentication flow defined by you. The following guide shows how to setup a simple passwordless authentication flow.
+The Auth category can be configured to perform a [custom authentication flow](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-challenge.html) defined by you. The following guide shows how to setup a simple passwordless authentication flow.
 
 ## Prerequisites
 
-import ios0 from "/src/fragments/lib/auth/ios/getting_started/10_preReq.mdx";
+import ios0 from '/src/fragments/lib/auth/ios/getting_started/10_preReq.mdx';
 
-<Fragments fragments={{ios: ios0}} />
+<Fragments fragments={{ ios: ios0 }} />
+
+import flutter0 from '/src/fragments/lib/auth/flutter/getting_started/10_preReq.mdx';
+
+<Fragments fragments={{ flutter: flutter0 }} />
 
 ## Configure Auth Category
 
-import ios1 from "/src/fragments/lib/auth/ios/signin_with_custom_flow/10_cli_setup.mdx";
+import ios1 from '/src/fragments/lib/auth/ios/signin_with_custom_flow/10_cli_setup.mdx';
+
+<Fragments fragments={{ ios: ios1 }} />
 
-<Fragments fragments={{ios: ios1}} />
+import flutter1 from '/src/fragments/lib/auth/flutter/signin_with_custom_flow/10_cli_setup.mdx';
+
+<Fragments fragments={{ flutter: flutter1 }} />
 
 ## Register a user
```

Comment:
When registering the user, even if my custom Auth logic uses a one-time password, I still need to use Username+password when registering a new user?

Reply:
Yes - noted elsewhere.